src/components/Newsletter/Newsletter.js

```diff
@@ -35,50 +42,131 @@ class Newsletter extends React.Component {
       });
   }
 
+  handleSelectChange = selectedOption => {
+    this.alertsSelectionArray = selectedOption;
+    this.setState({
+      alertSignUps: this.alertsSelectionArray
+        .map(item => item.full_fips_code)
+        .join(','),
+    });
+  };
+
   render() {
     const { stateId, county } = this.props;
+
+    const stateDataset = US_STATE_DATASET.state_dataset.map(state => {
+      return {
+        ...state,
+        full_fips_code: state.state_fips_code + '000',
+      };
+    });
+    const countyDataset = [];
+
+    each(US_STATE_DATASET.state_county_map_dataset, (value, key) => {
+      countyDataset.push(...value.county_dataset);
+    });
+
+    const autocomplete_options = stateDataset.concat(countyDataset);
+
     return (
       <StyledNewsletter>
         <form
           ref={f => (this.form = f)}
           className="js-cm-form"
           id="subForm"
           method="post"
-          data-id="2BE4EF332AA2E32596E38B640E90561930C9C3A433D015D9D4BD88E99175E51395EF5EBFFD527179E032AC15455BB1208D87A6CE87843E524B0EA520CBFF446E"
+          data-id="2BE4EF332AA2E32596E38B640E905619E90CD5DAC48A878CDEBFFE3B420D8CD24E4AEABAB52A4CE3526219C7A966AE965B84F99C823C89EFF1F01B28DE4F975E"
         >
           <input
             hidden
             readOnly
             aria-label="state"
             value={stateId || ''}
-            id="fieldjlkiyul"
-            maxLength="200"
-            name="cm-f-jlkiyul"
+            id="fieldjrdtwi"
+            maxlength="200"
+            name="cm-f-jrdtwi"
           />
           <input
             hidden
             readOnly
             aria-label="county"
+            id="fieldjrdtwd"
+            maxlength="200"
+            name="cm-f-jrdtwd"
             value={county || ''}
-            id="fieldjlkulhk"
-            maxLength="200"
-            name="cm-f-jlkulhk"
           />
```

Comment:
Do we still want / need these state / county custom fields anymore?  It seems like the new alert_list_csv field essentially supersedes them.

Reply:
i think it's probably useful to have the signup page idk it might break the form to remove them so i'm gonna leave